devenv/datasources.yaml

```
@@ -352,3 +352,10 @@ datasources:
     access: proxy
     url: http://localhost:3101
     editable: false
+
+  - name: JSON API - Near Misses
+    type: simpod-json-datasource
+    editable: true
+    url: http://0.0.0.0:8000/grafana/near_misses
```

Could we use port `8080` as this is the default SM port, and also `audacity` runs on `8000` already
```suggestion
    url: http://0.0.0.0:8080/grafana/near_misses
```

---

have pushed this change :)